### ISSUE TITLE
Updated: Sony Xperia Z4 release date news and rumors

### ISSUE THREAD

**Issue by ejebha** [ejebha/tech#7]
<b>Updated: Sony Xperia Z4 release date, news and rumors</b><br><div xmlns="http://www.w3.org/1999/xhtml">
<img alt="Updated: Sony Xperia Z4 release date, news and rumors" src="http://cdn.mos.techradar.com/art/mobile_phones/Sony/XperiaZ3/Hands%20on%203/Sony%20Xperia%20Z3%20review%20(13)-470-75.JPG"><br><h3>Xperia Z4: release date, design and screen</h3>
<br><p><strong>Latest Update:</strong> <em>The Sony Xperia Z4 has seemingly been revealed in a series of snaps and its camera has even been put to the test.</em></p>
<br><p>It might not seem that long since the <a href="http://www.techradar.com/reviews/phones/mobile-phones/sony-xperia-z3-1263309/review#articleContent">Xperia Z3</a> was released, but with Sony's rapid release schedule for its flagship smartphones the rumor mill has already been working overtime on the Sony Xperia Z4.</p>
<br><p>We're hearing plenty of rumors about Sony's next flagship, including <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-xperia-z4-could-be-world-s-most-powerful-smartphone-1269924">one report</a> which reckons the Japanese firm will return to an annual launch of its top device, ditching the recent trend of every six months, though not until after the Z4 launches.</p>
<br><p>A QHD display, an octa-core processor and an improved camera are all being touted too, so the Xperia Z4 should easily give the latest Samsung and HTC flagships a run for their money.</p>
<br><h3>Cut to the chase</h3>
<br><ul>
<li style=""><br></li>
<li>
<strong>What is it?</strong> Sony's next flagship smartphone</li>
<li style=""><br></li>
<li>
<strong>When is it out?</strong> Sometime in the summer probably</li>
<li style=""><br></li>
<li>
<strong>What will it cost?</strong> A lot - this will be a high-end device</li>
<li style=""><br></li>
</ul>
<br><h3>Sony Xperia Z4 release date</h3>
<br><p>Some <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-s-xperia-z4-might-not-launch-at-mwc-in-march-after-all-1282223">rumors suggest</a> that Sony is moving from two flagships a year to just one and pushing the Sony Xperia Z4 back to the summer.</p>
<br><p>On the other hand Sony has already started rolling out its '4' product range with the <a href="http://www.techradar.com/reviews/pc-mac/tablets/sony-xperia-z4-tablet-1286092/review">Xperia Z4 Tablet</a> and the <a href="http://www.techradar.com/reviews/phones/mobile-phones/sony-xperia-m4-aqua-1286091/review">Xperia M4 Aqua</a>, so it would make sense to have a smartphone alongside them. Plus, speaking to <a href="http://www.trustedreviews.com/news/sony-promises-xperia-z4-release-date-before-september">TrustedReviews</a>, Tim Harrison, Sony's Vice President of Global Communications and PR, claimed the company hasn't moved to a 12 month product cycle.</p>
<br><p>So maybe we won't have to wait too long after all. The Xperia Z4 Tablet isn't set to launch till June anyway, so that still gives Sony a few months to announce its next flagship and get it on the market at the same time.</p>
<br><h3>Sony Xperia Z4 design</h3>
<br><p>It looks like the Sony Xperia Z4 has been spotted in the wild, as snaps of a very Z4-looking-phone have been <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/see-the-sony-xperia-z4-s-camera-and-interface-in-latest-picture-leaks-1290227">posted by both Nowhereelse.fr and in2mobile</a>. In fact it also looks a lot like the Xperia Z3. That's not necessarily a bad thing, as the Xperia Z3's attractive OmniBalance design was one of its strongest features.</p>
<br><p><img class="zoomable" title="Credit: Nowhereelse.fr" width="420" alt="Xperia Z4" src="http://mos.futurenet.com/techradar/art/mobile_phones/Sony/Xperia%20Z4/Leaks/Xperia%20Z4%20NWE%20leak-420-90.jpg"></p>
<br><p>According to a source that contacted Phone Arena, the Xperia Z4 will have thinner top and bottom bezels as well as more rounded corners as you can sort of see in the accompanying image below.</p>
<br><p><img width="420" alt="Xperia Z4 leak" src="http://mos.futurenet.com/techradar/art/mobile_phones/Sony/Xperia%20Z4/Xperia%20Z4%20phonearena%20leak-420-100.jpg"></p>
<br><p>Images apparently taken from the email account of the Sony CEO during the major Sony hack also reveal <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/early-images-of-sony-xperia-z4-leak-with-a-little-help-from-007-1277841">early renders</a> of the Xperia Z4, showing a mirrored front and back.</p>
<br><p><img title="It's actually the 5th generation of Xperia Z (credit: Gizmodo)" width="420" alt="Sony Xperia Z4 - LEAK" src="http://mos.futurenet.com/techradar/art/mobile_phones/Sony/Xperia%20Z4/Leaks/Gizmodo-Sony/XperiaZ4-LEAK-Sony-01-420-90.jpg"></p>
<br><p>Annotated images that were included in the leak show that the Xperia Z4 could come with a 1.2mm glass panel and resin bezel, while another rumor suggested that Sony could be releasing a <a href="http://www.techradar.com/news/phone-and-communications/sony-may-please-music-loves-with-a-walkman-edition-of-the-xperia-z4-1280014">Walkman edition of the Xperia Z4</a>. This version would concentrate on music playing with hi-res audio codec support, two-day battery life and improved S-Force PRO front-facing speakers.</p>
<br><p><img width="420" alt="Sony Xperia Z4" src="http://mos.futurenet.com/techradar/art/mobile_phones/Sony/Xperia%20Z4/Leaks/Gizmodo-Sony/XperiaZ4-LEAK-Sony-04-420-90.jpg"></p>
<br><h3>Sony Xperia Z4 screen</h3>
<br><p>The Sony Xperia Z4 <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-s-xperia-z4-just-got-outed-warts-and-all-1289965">appears to have been benchmarked</a> with a 5.1-inch 1080 x 1920 screen, but there have been a lot of conflicting reports about its display. <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-xperia-z4-could-be-world-s-most-powerful-smartphone-1269924">Early rumors</a> hinted that the screen would be 5.5-inches, a fair bit larger than the 5.2-inch display on the Xperia Z3.</p>
<br><p>Another rumour suggested that the screen will be slightly smaller at 5.4-inches, though still an upgrade over the Z3.</p>
<br><p>Yet <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-xperia-z4-said-to-be-sticking-with-a-5-2-inch-screen-1275931">more rumors</a> claim that neither of those sizes is correct, and instead Sony will be sticking with a 5.2-inch display. That has now been backed up with details from a <a href="http://www.esato.com/board/viewtopic.php?topic=207716&amp;start=1309">new leak</a> that suggests there will be a 5.2-inch screen with a Quad HD resolution of 1440 x 2560 pixels, equalling 565ppi.</p>
<br><p><img class="zoomable" title="Credit: Nowhereelse.fr" width="420" alt="Xperia Z4" src="http://mos.futurenet.com/techradar/art/mobile_phones/Sony/Xperia%20Z4/Leaks/Xperia%20Z4%20NWE%20leak2-420-90.jpg"></p>
<br><p>The other detail that rumors seem to be conflicted on is regarding the resolution of the screen. Although some rumours claim that Sony is sticking with 1080p as the resolution, <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-xperia-z4-could-be-world-s-most-powerful-smartphone-1269924">other rumors</a> are suggesting it will get a substantial boost to QHD 1440 x 2560, the same as the <a href="http://www.techradar.com/reviews/phones/mobile-phones/lg-g3-1250581/review">LG G3</a>.</p>
<br><p>Perhaps most interestingly we've also heard rumors that both these resolutions are correct, and that <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-plotting-two-flavours-of-the-xperia-z4-says-report-1279517">Sony will release two different versions of the Xperia Z4</a>.</p>
<br><h3>Xperia Z4: Rivals, power, camera and more</h3>
<br><h3>Sony Xperia Z4 rivals</h3>
<br><p>Sony's biggest challenges will come from the <a href="http://www.techradar.com/reviews/phones/mobile-phones/samsung-galaxy-s6-1285636/review">Samsung Galaxy S6</a> and the <a href="http://www.techradar.com/reviews/phones/mobile-phones/htc-one-m9-1285623/review">HTC One M9</a>. Two big name, high spec devices which have a head start on the Xperia Z4.</p>
<br><p><img width="420" alt="Galaxy S6" src="http://mos.futurenet.com/techradar/art/mobile_phones/Samsung/Galaxy%20S6/Full%20review/Hands%20on/Samsung_Galaxy_S6_review%20(16)-420-90.JPG"></p>
<br><p>But it will also have to face up to the <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/lg-g4-what-we-want-to-see-1266879">LG G4</a>, which may release at a similar time and may also deliver a similarly powerful camera. Even the <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/oneplus-two-what-we-want-to-see-1288582">OnePlus 2</a> could be a tempting alternative, as it's likely to offer similar specs at a lower price.</p>
<br><h3>Sony Xperia Z4 camera and battery</h3>
<br><p>A snap supposedly taken on the Xperia Z4 <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-xperia-z4-camera-shot-proves-it-s-ready-for-the-wild-1290351">has been uploaded</a> and it suggests a 20.7MP camera, just like the Xperia Z3.</p>
<br><p><img width="420" alt="Xperia Z4 camera shot" src="http://mos.futurenet.com/techradar/art/mobile_phones/Sony/Xperia%20Z4/Leaks/Sony%20Xperia%20Z4-420-90.jpg"></p>
<br><p>Other rumors echo that, but they add that it may use a new sensor capable of taking in more light, along with an upgraded 4.8MP front-facing snapper. A <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-s-xperia-z4-just-got-outed-warts-and-all-1289965">benchmark</a> largely supports that, reporting a 19MP rear sensor and a 4.8MP front-facing one.</p>
<br><p><img class="zoomable" title="Credit: in2mobile" width="420" alt="Xperia Z4" src="http://mos.futurenet.com/techradar/art/mobile_phones/Sony/Xperia%20Z4/Leaks/Xperia%20Z4%20in2mobile%20leak-420-90.jpg"></p>
<br><p><a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-xperia-z4-may-pack-in-an-even-more-impressive-camera-1272728">Other leaks</a> suggest the Z4's camera will feature the IMX189AEG CMOS image sensor. This sensor will be able to record 2K video (2048 x 1080) at 16,000 frames per second, with 4K video (6144 x 2160) recording at 240 frames per second, much faster (and therefore smoother) than the Z3.</p>
<br><p>Current rumors peg the battery at <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-xperia-z4-camera-shot-proves-it-s-ready-for-the-wild-1290351">3400mAh</a>, which is a slight boost over the already-long-lasting 3100mAh Xperia Z3.</p>
<br><h3>Sony Xperia Z4 OS and power</h3>
<br><p>We could be in for something quite special with the Xperia Z4, as a <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-xperia-z4-secrets-revealed-here-s-what-to-expect-1284127">benchmark result</a> pegs it as having a Snapdragon 810 processor and 3GB of RAM and that's since been echoed by <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-s-xperia-z4-just-got-outed-warts-and-all-1289965">another benchmark</a>, which also revealed an Adreno 430 GPU and 32GB of storage.</p>
<br><p>Earlier claims include specs which largely line up with this. A 64-bit 2.0GHz octa-core Snapdragon 810 processor, 4GB of RAM, Adreno 430 GPU, 32GB of storage and an Exmor RS CMOS sensor have all been touted for the handset. It will supposedly support Bluetooth 4.1 and LTE Cat.6 connectivity, leaving it equipped to handle LTE-A speeds.</p>
<br><p><a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-xperia-z4-said-to-be-sticking-with-a-5-2-inch-screen-1275931">We've seen other rumors</a> that offer conflicting reports on the chip that will power the Xperia Z4 though, with claims that it will use the slightly weaker Snapdragon 805. Either way, both these processors offer an upgrade over the Snapdragon 801, which is found in the Xperia Z3.</p>
<br><p>We've heard relatively little about what operating system the Xperia Z4 will run, though it's a safe bet that it will use Sony's custom overlay. There's little doubt that it will also run <a href="http://www.techradar.com/reviews/pc-mac/software/operating-systems/android-5-0-lollipop-1271651/review">Android 5.0 Lollipop</a>, especially as Sony has already rolled that out to some of its phones.</p>
<br><h3>Sony Xperia Z4 cost</h3>
<br><p>There's been no information on how much the Xperia Z4 will cost, however as it's a flagship smartphone (and a Sony one at that) we'd expect it to ship with a high price tag.</p>
<br><p>The Xperia Z3 launched at &pound;549 ($630, AU$849) and we'd expect the Z4 to launch at around the same price.</p>
<br><p>Rumors have also suggested that <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-plotting-two-flavours-of-the-xperia-z4-says-report-1279517">Sony could be plotting two versions of its flagship smartphone</a>. There would be a Z4 with a 1080 x 1920 full high definition screen, similar to the one found on the Xperia Z3. The second version would offer a much higher 1440 x 2560 (QHD) resolution.</p>
<br><p>These rumors claim that the 1080p version will cost less than the QHD version, though we'd take those claims with a pinch of salt.</p>
<br><h3>Xperia Z4: what we want to see</h3>
<br><p>We know the Xperia Z4 is on its way and over the last few months we've been making a wishlist of what we want to see.</p>
<br><h3>4K video that actually works</h3>
<br><p>Sony's been trumpeting its phones 4K video capabilities since the Xperia Z2, yet neither that nor the Xperia Z3 can actually record for more than around five minutes without heating up and shutting down the video.</p>
<br><p><img width="420" alt="Z3 camera" src="http://mos.futurenet.com/techradar/art/mobile_phones/Sony/XperiaZ3/Sony%20Xperia%20Z3%20hands%20on/Sony_Xperia_Z3_review%20(17)-420-100.JPG"></p>
<br><p>That's obviously a major problem and while it doesn't quite render the 4K feature redundant it does severely limit it. The fact that Sony has left it unfixed in the Xperia Z3 is perplexing, presumably there isn't an easy fix, but if longer videos were possible it could be a genuinely useful feature, whereas right now it's just kind of an embarrassment.</p>
<br><p>So hopefully it will be fixed for the Sony Xperia Z4. You never know, some people might even have 4K televisions by then too.</p>
<br><h3>A QHD display</h3>
<br><p>The <a href="http://www.techradar.com/reviews/phones/mobile-phones/sony-xperia-z-1119637/review">Sony Xperia Z</a>, <a href="http://www.techradar.com/reviews/phones/mobile-phones/sony-xperia-z1-1177393/review">Z1</a> and Z2 all had 1080p displays and that was fine as that was the best any smartphone had at the time and the screen size was at least being increased.</p>
<br><p>But then the Xperia Z3 also launched with a 1080p screen and that was slightly more disappointing, since the size remained the same and other than increased brightness there wasn't much of an upgrade, plus some other phones, such as the <a href="http://www.techradar.com/reviews/phones/mobile-phones/lg-g3-1250581/review">LG G3</a>, have started including QHD displays.</p>
<br><p>We want the Xperia Z4 to wow us and a 1080p display for the fifth time in a row won't do that, so we're hoping it's QHD.</p>
<br><h3>Smaller bezels</h3>
<br><p><img width="420" alt="Xperia Z3" src="http://mos.futurenet.com/techradar/art/mobile_phones/Sony/XperiaZ3/Hands%20on%203/Sony%20Xperia%20Z3%20review%20(14)-420-100.JPG"></p>
<br><p>The Sony Xperia Z3 looks great, but the bezels above and below the screen are off-puttingly large. Presumably there's a reason for their size, they're likely housing vital components required to make the phone actually work, but they're unsightly and we hope they're smaller next time around.</p>
<br><h3>Improved low-light performance</h3>
<br><p>For the most part the Xperia Z3 has a perfectly capable and even impressive camera, but its low-light performance could still use some work.</p>
<br><p>Sony made some strides with the Xperia Z3, improving it over the Z2, but seemingly more with software than hardware as its low-light photos tend to come out over-processed, leaving it trailing behind the likes of the <a href="http://www.techradar.com/reviews/phones/mobile-phones/iphone-6-1264565/review">iPhone 6</a> and the <a href="http://www.techradar.com/reviews/phones/mobile-phones/htc-one-m8-1235307/review">HTC One M8</a> in darker scenes.</p>
<br><p>Clearly Sony is aware of the problem given that it made some improvements on the Xperia Z3, but it didn't go far enough. We want proper hardware improvements, rather than awkward software fixes.</p>
<br><p>Sony is due to release a flagship mobile image sensor, the <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-xperia-z4-looks-set-for-mwc-1273223">Exmor RS IMX230</a> in April 2015 and it is likely that the Xperia Z4 will come with the new sensor.</p>
<br><h3>More power</h3>
<br><p>With a 2.5GHz quad-core Snapdragon 801 processor and 3GB of RAM you might not think the Xperia Z3 was lacking for power, yet in some ways it is. For one thing all that power seemingly hasn't translated into performance, as surprisingly we found a little bit of lag in use, especially when switching between apps.</p>
<br><p><img width="420" alt="Z3" src="http://mos.futurenet.com/techradar/art/mobile_phones/Sony/XperiaZ3/Hands%20on%203/Sony%20Xperia%20Z3%20review%20(12)-420-100.JPG"></p>
<br><p>It's not a big deal but it does feel less snappy than most other recent flagships despite the specs, which on paper should be more than up to the task. So we're more after increased performance than an actual boost in specs, if it takes improved specs to do that then fine, but if the same thing can be achieved with a bit more optimisation that's fine too.</p>
<br><p>Having said that superior specs wouldn't be a bad thing either way, as the Snapdragon 801 is starting to look a bit long in the tooth with the <a href="http://www.techradar.com/reviews/phones/mobile-phones/samsung-galaxy-note-4-1263301/review">Samsung Galaxy Note 4</a> using a Snapdragon 805 chip and the Snapdragon 810 already hitting the shelves.</p>
<br><h3>Fewer bugs</h3>
<br><p>Another surprising aspect of the Xperia Z3 was just how buggy it seemed, with the keyboard sometimes bouncing or proving unresponsive and the notifications bar occasionally getting stuck half way down.</p>
<br><p>Hopefully Sony will patch those bugs out of the Xperia Z3 anyway but we'd really rather it makes sure the Xperia Z4 is bug-free before launching it.</p>
<br><h3>Faster camera and autofocus</h3>
<br><p>As long as the light's good the Xperia Z3 can take some exceptional snaps, but one problem with it is actually getting to the point of being able to take a photo, as both launching the camera and the autofocus are a little slower than we'd like.</p>
<br><p>We're talking milliseconds here, but that can still mean the difference between capturing a moment and missing it, so hopefully the Xperia Z4 will have a snappier snapper.</p>
<br><p>The new Exmor RS IMX230, which the Xperia Z4 is likely to come with, is claimed by Sony to be the industry's first CMOS image sensor built for smartphones which can track fast-moving subjects thanks to built-in image plane phase detection autofocus.</p>
<br><p>This new sensor would make the Xperia Z4's camera much better at photographing fast moving subjects.</p>
<br><h3>A new design</h3>
<br><p>We've mentioned that we want smaller bezels but what we'd really like to see is a totally new design. Sony has done a good job of refining its OmniBalance design but there's only so much you can do and with handset after handset sporting fundamentally the same look it's starting to get a bit boring.</p>
<br><p><img width="420" alt="Z3 side" src="http://mos.futurenet.com/techradar/art/mobile_phones/Sony/XperiaZ3/Hands%20on%203/Sony%20Xperia%20Z3%20review%20(2)-420-100.JPG"></p>
<br><p>Not to mention the fact that for our money both the iPhone 6 and the HTC One M8 look better. So hopefully Sony will do something new and exciting with the Xperia Z4.</p>
<br><p>Make it metal, give it curves, make it triangular. Ok, maybe not that last one, but we're sure Sony can come up with something, oh and get rid of all those annoying flaps! Most likely it will stick with OmniBalance, but we can dream.</p>
<br><h3>Better Lifelogging</h3>
<br><p>Sony's Lifelogging app is an interesting idea, as it goes beyond just counting steps to also tell you how much running, cycling and sleeping you do and even how long you spend listening to music or watching videos (as long as you do it on your phone).</p>
<br><p>Essentially as the name suggests it's designed to log your life, but it could do with some improvements. The biggest issue is that it's just not accurate enough, but there are also quirks like the fact that it counts all apps as games, so even if you've never touched a game on your smartphone it will likely peg you as a big mobile gamer.</p>
<br><p>Plus, while all this information is interesting it doesn't really lead to anything or give you any insights based on the data. If Sony can work on all those things for its Z4 implementation then it could become genuinely useful.</p>
<br><h3>Innovation</h3>
<br><p>Asking for innovation might seem a bit vague, but with smartphones increasingly levelling out when it comes to specs it's innovation that can really help them stand out, so it would be great if the Sony Xperia Z4 had one or more innovative new features.</p>
<br><p>What those might be is anyone's guess, maybe a scratch proof body, maybe a virtual reality display, probably not a virtual reality display, but something and ideally something that's more than just a gimmick.</p>
<br><p>Come on Sony, give us a new feature that even Z3 owners will consider upgrading for.</p>
<br><ul>
<li style=""><br></li>
<li>We have <a href="http://www.techradar.com/news/phone-and-communications/mobile-phones/lg-g4-what-we-want-to-see-1266879">ideas for the LG G4</a> as well.</li>
<li style=""><br></li>
</ul>
<br><br><br><br><br>
from TechRadar: All latest feeds http://www.techradar.com/news/phone-and-communications/mobile-phones/sony-xperia-z4-what-we-want-to-see-1268100?src=rss&amp;attr=all<br><br>
via <a href="https://ifttt.com/?ref=da&amp;site=blogger">IFTTT</a>
</div>
<br><br>
via Blogger http://www.tfrwih.com/2015/04/updated-sony-xperia-z4-release-date_13.html<br>
April 13, 2015 at 12:23PM
